src/frontend.cpp

```diff
@@ -42,6 +42,7 @@
 #include "config_strings.h"
 #include "config_campaigns.h"
 #include "config_creature.h"
+#include "config_spritecolors.h"
```

Comment:
header not needed since you didn't need the function
other then that looks ok to me